src/core/send-commercial-metrics.ts

```diff
@@ -143,6 +143,21 @@ const getOfflineCount = (): Metric[] =>
 		  ]
 		: [];
 
+/**
+ * Measures added with @guardian/libs’s startPerformanceMeasure API
+ * that are assigned to the 'dotcom' team.
+ *
+ * Allows for more granular monitoring of web page performance.
+ */
+const getDotcomPerformanceMeasures = (): Metric[] =>
+	window.performance
+		.getEntriesByType('measure')
+		.filter(({ name }) => name.startsWith('doctom:'))
```

Comment:
Before we start sending data to the lake a PR to consider:

https://github.com/guardian/csnx/pull/756

Reply:
Thinking about https://github.com/guardian/csnx/pull/760, actually!